src/framework/theme/components/icon/icon-libraries.ts

```diff
@@ -29,10 +29,16 @@ function throwWrongPackTypeError(name: string, type: string, desiredType: string
 
 /**
  * This service allows to register multiple icon packs to use them later within `<nb-icon></nb-icon>` component.
+ *
+ * `registerFontPack` supports fonts with ligatures, config should contain ligature flag
+ * ```ts
+ * this.iconLibraries.registerFontPack("material-icons", {
+ *   ligature: true,
+ * });
+ * ```
```

Comment:
Let's expand `NbIconLibraries` docs a little. Structure it as:
1. registering svg pack
2. registering font pack
3. registering ligature font pack
Add 1 and 2 similar to already existing 3rd section.